packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1930,6 +1966,61 @@ export class ContainerRuntime
 		this.skipSavedCompressorOps = pendingRuntimeState?.pendingIdCompressorState !== undefined;
 	}
 
+	private async loadAndInitializeRuntimeAttributor(context: IContainerContext): Promise<void> {
+		if (this.runtimeAttributorP === undefined) {
+			this.runtimeAttributorP = this.getDelayLoadedRuntimeAttributor();
+		}
+		await this.runtimeAttributorP
+			.then(async (attributor) => {
+				this.runtimeAttributorModuleLoaded = true;
```

Comment:
Is it too early to set it, given async nature of the code below?
Should it be more set at the end?